lib/foreman_tasks/tasks/export_tasks.rake

```diff
@@ -253,8 +255,20 @@ namespace :foreman_tasks do
         total = tasks.count
 
         tasks.each_with_index do |task, count|
-          File.open(File.join(tmp_dir, "#{task.id}.html"), 'w') { |file| file.write(PageHelper.pagify(renderer.render_task(task))) }
-          puts "#{count + 1}/#{total}"
+          begin
+            File.open(File.join(tmp_dir, "#{task.id}.html"), 'w') { |file| file.write(PageHelper.pagify(renderer.render_task(task))) }
+            puts "#{count + 1}/#{total}"
+          rescue StandardError => e
+            puts "WARNING: task failed to export. Additional task details below."
+            puts "#{task.inspect}"
```

Comment:
`whatever.inspect` already returns a string, so wrapping it in another string and then using string interpolation is a bit redundant

Reply:
Resolved.